### PR TITLE
Fix/tpfs source

### PR DESCRIPTION
**Pull request by DShau18**
Issue: https://issues.cask.co/browse/CDAP-10269
Build: https://builds.cask.co/browse/HYP-BAD327-4

### CODE REVIEW

File: core-plugins/pom.xml [hunk 1]
```diff
@@ -239,6 +245,58 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-core_2.10</artifactId>
```
Comment (albertshau) [inline core-plugins/pom.xml]: why was spark added?

File: core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLBatchTestBase.java [hunk 2]
```diff
@@ -166,7 +166,7 @@ protected void cleanPartitions(TimePartitionedFileSet fileSet) throws IOExceptio
       if (locName.endsWith(".parquet")) {
         Path parquetFile = new Path(file.toString());
         AvroParquetReader.Builder<GenericRecord> genericRecordBuilder = AvroParquetReader.builder(parquetFile);
-        ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
+        org.apache.parquet.hadoop.ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
```
Comment (albertshau) [inline core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLBatchTestBase.java]: how come the full name is being used? Can we just remove the parquet.hadoop.ParquetReader import and import this class?

File: core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLSnapshotTestRun.java [hunk 3]
```diff
@@ -295,7 +295,7 @@ private void testSource(String sourceETLPlugin, String sourceName, Map<String, I
       } else if (fileName.endsWith(".parquet")) {
         Path parquetFile = new Path(file.toString());
         AvroParquetReader.Builder<GenericRecord> genericRecordBuilder = AvroParquetReader.builder(parquetFile);
-        ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
+        org.apache.parquet.hadoop.ParquetReader<GenericRecord> reader = genericRecordBuilder.build();
```
Comment (albertshau) [inline core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLSnapshotTestRun.java]: same question about the full class name

File: core-plugins/pom.xml (hunk 1 above)
Comment (DShau18) [reply core-plugins/pom.xml]: it was added to exclude parquet-hadoop on line 288.

Comment (albertshau) [reply core-plugins/pom.xml]: you shouldn't add a new dependency to exclude a transitive one. If you need to exclude a transitive one, it should be excluded from the dependency that pulls it in. For example, if this is getting pulled in by cdap-unit-test, it should be excluded from there.

Comment (albertshau) [reply core-plugins/pom.xml]: Also, why does it need to be excluded? Does it cause test failures?

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**